noir-projects/noir-protocol-circuits/crates/private-kernel-lib/src/private_kernel_reset.nr

```diff
@@ -182,14 +182,19 @@ mod tests {
             let mut previous_kernel = FixtureBuilder::new().in_vk_tree(PRIVATE_KERNEL_INNER_INDEX);
             previous_kernel.set_first_nullifier();
 
+            let note_hash_read_request_hints_builder =
+                unsafe { NoteHashReadRequestHintsBuilder::new() };
+            let nullifier_read_request_hints_builder =
+                unsafe { NullifierReadRequestHintsBuilder::new() };
```

Comment:
Why are we making construction of these unconstrained?

Reply:
Ooops, there were calls to unconstrained in the construction at some point but then I removed them. Will change it back! Good spot!